.gitignore

```diff
@@ -11,4 +11,5 @@ dist/
 *.egg-info
 htmlcov/
 .coverage
-test-results.xml
\ No newline at end of file
+test-results.xml
+.python-version
```

Comment:
did you mean to commit your .gitignore?

Reply:
yes

Reply:
why? haven't had any issue with this.
do you have a specific issue?